doc/benchmark.html

```diff
@@ -1555,13 +1555,13 @@
     html += "<tr>";
     html += "<th></th>";
     html += "<th style='text-align: left;'>Query</th>";
-    for (j in filtered_results) {
+    for (j = 0; j < filtered_results.length; j++) {
```

Comment:
It seems that both variant work properly in this context. But local `var` is really better.